src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -286,14 +288,142 @@ def create_certificate(client, vault_base_url, certificate_name, certificate_pol
 
 create_certificate.__doc__ = KeyVaultClient.create_certificate.__doc__
 
+def add_certificate_contact(client, vault_base_url, contact_email, contact_name=None,
+                            contact_phone=None):
+    """ Add a contact to the specified vault to receive notifications of certificate operations. """
```

Comment:
I don't think we have a recommended way honestly. You *have* to use the help file for group help, but not for command help. IMO, this is ideally where it would be because you have the benefit of locality.